tasks/agent.yml

```diff
@@ -66,6 +66,19 @@
   with_items:
     "{{ bamboo_agent.capabilities }}"
   when: (item.source is defined and item.source == "repository")
```

Comment:
I think this line should be removed.

Reply:
Could you elaborate on that? That line is not part of my change, actually.

Reply:
Yea, this is my code, but you've added a 2nd "when" directive on the next line. I think this will not work :)

Reply:
Ah, sorry, I see what you mean ;)